sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/IndexZPageHandler.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.extensions.zpages;
+
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+
+final class IndexZPageHandler extends ZPageHandler {
+  private static final String INDEX_URL = "/";
+  private static final String INDEX_NAME = "Index";
+  private static final String INDEX_DESCRITION = "Index page of zPages";
+  private static final Logger logger = Logger.getLogger(IndexZPageHandler.class.getName());
+  @Nullable private final List<ZPageHandler> availableHandlers;
+
+  IndexZPageHandler(@Nullable List<ZPageHandler> availableHandlers) {
```

Comment:
We should almost never have a nullable container, if there are no handlers the caller would just pass empty

Reply:
Should I make adjustment and open another PR?

Reply:
Yeah I think that'd be nice :)